README.md

```diff
@@ -61,3 +61,28 @@ print(csse.loc[('CZ', "2020-03-28")])
 * We enforce [black](https://github.com/psf/black) formatting (with the default style).
 * Use `pytest` for testing, add tests for your code!
 * Use pull requests for both this and the data repository.
+
+
+## Running pipeline to get web export
+Assuming you've installed deps via `poetry install` in the root epimodel repo.
+
+0. clone data repo: `git clone https://github.com/epidemics/epimodel-covid-data data`
+1. `./do update_john_hopkins`
+2. add a Foretold token into `config.yaml` in `foretold_channel` and run `./do update_foretold`
+3. **TODO?: Run gleamviz and get batch file? What's being fetched inside the file?** 
+4. having the Gleam Batch import file:
+    ```
+    ./do import_gleam_batch data/batch-2020-04-03T23-35-24.482054+02-00.hdf5
```

Comment:
Here I am getting:
```
./do import_gleam_batch data/batch-2020-04-03T23-35-24.482054+02-00.hdf5
INFO:epimodel.regions:Name index has 7 potential conflicts: ['american samoa', 'georgia', 'guam', 'northern mariana islands', 'puerto rico', 'united states minor outlying islands', 'virgin islands (u.s.)']
Traceback (most recent call last):
  File "./do", line 137, in <module>
    main()
  File "./do", line 133, in main
    args.func(args)
  File "./do", line 72, in import_batch
    batch.import_sims(
  File "/home/dan/prac/rats/epimodel/epimodel/gleam/batch.py", line 82, in import_sims
    raise Exception(f"Would overwrite existing `new_fraction` in {self}!")
Exception: Would overwrite existing `new_fraction` in <Batch data/batch-2020-04-03T23-35-24.482054+02-00.hdf5 24 sims, 281520 rows>!
Closing remaining open files:data/batch-2020-04-03T23-35-24.482054+02-00.hdf5...done
```
please help.

Reply:
When I set `overwrite=True`, I get:
```
INFO:epimodel.regions:Name index has 7 potential conflicts: ['american samoa', 'georgia', 'guam', 'northern mariana islands', 'puerto rico', 'united states minor outlying islands', 'virgin islands (u.s.)']
Traceback (most recent call last):
  File "./do", line 138, in <module>
    main()
  File "./do", line 134, in main
    args.func(args)
  File "./do", line 72, in import_batch
    batch.import_sims(
  File "/home/dan/prac/rats/epimodel/epimodel/gleam/batch.py", line 87, in import_sims
    raise Exception(f"No gleam result found for {sid} {sim.Name!r}")
Exception: No gleam result found for 1585949739678.574 'WEAK seasonality<br>WEAK reduction in air travel'
Closing remaining open files:data/batch-2020-04-03T23-35-24.482054+02-00.hdf5...done
```

Reply:
OK, I assume I have it wrong and the file you gave me is _after_ the import already (eye balling from the code). @gavento